source/03_required_vlrs.txt

```diff
@@ -106,22 +106,22 @@ complete description can be found in the GeoTIFF format specification. Here is
 a summary from a programmatic point of view for someone interested in
 implementation.
 
-The GeoKeyDirectoryTag is defined as an array of unsigned short values.
+The GeoKeyDirectoryTag is defined as an array of uint16_t values.
 Programmatically, the data can be structured as follows:
 
 ::
 
     struct sGeoKeys {
-        unsigned short wKeyDirectoryVersion;
-        unsigned short wKeyRevision;
-        unsigned short wMinorRevision;
-        unsigned short wNumberOfKeys;
+        uint16_t wKeyDirectoryVersion;
```

Comment:
I would suggest that we not change the definitions of these structs. They are coming directly from the GeoTIFF specification, so we do not want to introduce opportunity for a discrepancy.

Reply:
Hmm, aren't the GeoTIFF structs defined in terms of TIFF's `BYTE`/`SHORT`/`LONG` etc., rather than C's `char`/`unsigned short`/`unsigned long`? As far as I can tell, the basic TIFF data types are clearly defined in a way that aligns with C99's `uint8_t`/`uint16_t`/`uint32_t` etc. Thus, a discrepancy is already there IMO - between similarly-named TIFF and C data types.

Reply:
As long as the names represent the same thing, I don't see an issue.